src/components/AccountConfigDialog/ConnectAccountConfigDialog.jsx

```
@@ -0,0 +1,593 @@
+
+  return { isPolling };
+};
+
+const Trigger = ({ formData, serviceOrgName }) => {
+  return (
+    <Box width={"100%"} display={"flex"} flexDirection={"column"} gap="10px">
+      <Stack direction="row" alignItems="center" gap="16px">
+        <Box
+          sx={{
+            border: "1px solid #E4E7EC",
+            boxShadow: "0px 1px 2px 0px #1018280D",
```

Biome (1.9.4)

[error] This property value named boxShadow is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named boxShadow


(lint/suspicious/noDuplicateObjectKeys)